pkg/crc/config/config.go

```diff
@@ -239,11 +239,20 @@ func (c *Config) Get(key string) SettingValue {
 	}
 	return SettingValue{
 		Value:     value,
-		IsDefault: reflect.DeepEqual(setting.defaultValue, value),
```

Comment:
Good catch!
There's a 
```
case Path, string:
		value = cast.ToString(value)
```
in this function, did you try changing it to:
```
case Path: value = Path(cast.ToString(value))
```
?

Reply:
Yes tried this now, and it also works, updated to use this instead..